iOSNativeLoginTemplate/iOSNativeLoginTemplate/NativeLoginView.swift

```diff
@@ -155,8 +268,128 @@ struct NativeLoginView: View {
                 SalesforceManager.shared.nativeLoginManager().fallbackToWebAuthentication()
             }
         }.background(Gradient(colors: [.blue, .cyan, .green]).opacity(0.6))
-            .blur(radius: self.isAuthenticating ? 2.0 : 0.0)
+            .blur(radius: isAuthenticating ? 2.0 : 0.0)
     }
+    
+    ///
+    /// Submits the OTP delivery request when the request button is tapped.  This submits a request to the
+    /// `init/passwordless/login`endpoint and opens the submit OTP verification view on success.
+    ///
+    private func onRequestOtpTapped() {
+        // Reset the error message if needed.
+        errorMessage = ""
+        
+        // Show the progress indicator.
+        isAuthenticating = true
+        
+        // Execute for a new reCAPTCHA token.
+        reCaptchaClientObservable.reCaptchaClient?.execute(withAction: .login) {reCaptchaExecuteResult, error in
+            
+            // Guard for the new reCAPTCHA token.
+            guard let reCaptchaToken = reCaptchaExecuteResult else {
+                SalesforceLogger.e(AppDelegate.self, message: "Could not obtain a reCAPTCHA token due to error with description '\(error?.localizedDescription ?? "(A description wasn't provided.)")'.")
```

Comment:
I suppose since this is a struct that's why `NativeLoginView.self` cannot be used here.  It needs a class.  Is there any work-around for that and is it significant?

Reply:
Maybe we could add an extension for structs? Out of scope for this but could add it to the backlog, seems like a good modernization item